HCCGo/app/js/NotifierService.js

```diff
@@ -0,0 +1,102 @@
+
+notifierModule = angular.module('NotifierModule', [])
+
+notifierModule.factory('notifierService',['$log', '$q', '$routeParams', 'toastr', 
+                                           function($log, $q, $routeParams, toastr) {
+  
+   const {ipcRenderer} = require('electron');
+   const notifier = require('node-notifier');
+   var path = require('path');
+   var fs = require('fs');
+   
+   const toastrOptions = {closeButton: true,
+                          timeOut: 5000,
+                          extendedTimeOut: 5000,
+                          progressBar: true};
+   
+   /**
+   * To notify user of application dependent on window focus
+   *
+   */
+   
+  // Get window focus status
+  var getWinFocus = function() {
+    var deferred = $q.defer();
+	  ipcRenderer.on('focus-check-message', (event, arg) => {
```

Comment:
Yeah, maybe it is now accepted.  But it certainly is coffeescript.  Most of electron is written in coffeescript, so be careful when you take their examples.